### PR TITLE
Feat/CLET-141 migrate stream docs [MINOR]

### PR DESCRIPTION
**Pull request by diazproduction**
- Added docs for Stream.

#### Steps to test

- Go to the website project.
- Run `env DOCS_BRANCH=this-pr-branch yarn docs` to download the docs files from this PR.
- Run `yarn start` and go to `/docs` to check the changes.

### CODE REVIEW

**Comment by cornelfd** [comment]
I think we are missing a few pages (or I'm missing them in the PR):
https://transaction-stream.fidel.uk/docs/mobile-sdks-changelog
https://transaction-stream.fidel.uk/docs/web-sdk-changelog

File: stream/web-sdk/v3.md [hunk 1]
```diff
@@ -0,0 +1,744 @@
+# Web SDK
```
Comment (MariaJoaoMiraPaulo) [inline stream/web-sdk/v3.md:1]: We are missing the web-sdk changelog https://transaction-stream.fidel.uk/docs/web-sdk-changelog.. unless we move it to a different place, are we?

File: stream/cards.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Card Linking
 
-To be able to participate in your program, users must link their cards to it first. You can implement this in your application using Fidel API's SDKs and APIs.
+To be able to participate in your program, users must link their cards to it first. You can implement this in your application using Fidel API's [SDKs](/stream/sdks) and APIs.
```
Comment (cornelfd) [inline stream/cards.md:5]: Will the link work if it’s written like this? Should it not be a full link, like I see for the other links in the pages? (ex: `https://fidelapi.com/docs/stream/sdks`)

Comment (krisztina-nagy) [reply stream/cards.md:5]: Relative links should work, but you are right, the relative links I added were incorrect.
To fix it, I would need to know the exact path and file name of the page that describes the SDKs.
Are you planning on adding a page where we have some text regarding what is in the "SDKs" chapter's subpages? What will be the full path of that file?

Comment (krisztina-nagy) [reply stream/cards.md]: Relative links should work, but the relative links I added were incorrect, thanks for bringing it to my attention, fixed now.

**Review by pedroalmeida-fidel — APPROVED** [review]

**Review by cornelfd — APPROVED** [review]

**Pull request merged**